src/components/CallView/shared/LocalVideo.vue

```diff
@@ -295,7 +295,7 @@ export default {
 			handler: function(localStreamVideoError) {
 				if (localStreamVideoError) {
 					showError(t('spreed', 'Error while accessing camera'), {
-						timeout: TOAST_PERMANENT_TIMEOUT,
+						timeout: TOAST_DEFAULT_TIMEOUT,
```

Comment:
Instead of hardcoding to default, remove the timeout. But also here I would prefer to not timeout and just remove it when it now works successfully

Reply:
Sadly I see this message repeatedly when the camera is disabled, this is why I wanted it to go away.
Would need to investigate more in detail to see why sometimes it says "camera was not allowed" and sometimes "Error while accessing camera". From what I remember, if I disallow the camera the first time after seeing the dialog, the error is specific to that. However, next time when I open the browser it will fail.

Needs further research

Reply:
Well, `timeout: TOAST_DEFAULT_TIMEOUT,` is not needed, so just remove the option for now